guides/common/modules/con_facts-and-nic-filtering.adoc

```diff
@@ -0,0 +1,17 @@
+[id="facts-and-nic-filtering_{context}"]
+= Facts and NIC filtering
+
+Facts are host parameters such as total memory, operating system version, or architecture.
+You can find facts in *Monitor* > *Facts* and search hosts through facts or use facts in templates.
+
+In {Project}, collect facts from following various sources:
+
+* `subscription manager`
+* `ansible`
+* `puppet`
+
+{Project} is an inventory system for hosts and network interfaces.
+For hypervisors or container hosts, adding thousands of interfaces per host and updating the inventory every few minutes is inadequate.
+For each individual NIC reported, {Project} creates a NIC entry and those entries are never removed from the database.
+Parsing all the facts and comparing all records in the database makes {Project} extremely slow and unusable.
+To optimize the performance of various actions, most importantly fact import, you can use the options available in *Administer* > *Settings* > *Facts*.
```

Comment:
"Facts" is not a subpage of settings, but rather a tab. This should IHMO be rephrased.